data/uber.txt

```diff
@@ -331,6 +399,48 @@ Jolly Nature
 - Stone Edge
 - Iron Tail
 
+
+=== [ubers] Latios ===
+
+Latios @ Soul Dew
+Ability: Levitate
+EVs: 4 Def / 252 SpA / 252 Spe
+Timid Nature
+- Draco Meteor
+- Psyshock
+- Defog
+- Hidden Power [Fire] / Memento / Recover
+
+Latios @ Soul Dew
+Ability: Levitate
+EVs: 4 Def / 252 SpA / 252 Spe
+Timid Nature
+- Draco Meteor
+- Psyshock
+- Calm Mind
```

Comment:
Uh, slashing Defog and Calm Mind would result in the same thing as having two sets...